### ISSUE TITLE
uploading files to test galaxy failed

### ISSUE THREAD

**Issue by MingChen0919** [bcuser30/ML_GalaxyQueue#3]
INSTRUCTIONS: The following template is meant to provide the information that will help diagnose and reproduce your issue. Follow the directions below to complete the template. If the template is not appropriate for your issue you may remove it and describe your issue.

###System information
Please enter the following information (if able). 

**Galaxy Instance: test** 

### Issue description 
Uploading files to the test galaxy results in the error: **The cluster DRM system terminated this job**.
I tried several file types: csv, gff3, bam. This error is probably not file type specific.

<img width="410" alt="screen shot 2018-05-30 at 1 52 38 pm" src="https://user-images.githubusercontent.com/1262709/40738362-5cb76c3e-6411-11e8-9f70-bec605a1292f.png">

**Comment by heh30** [comment]
Are you using the navigation of 'Get Data - Upload File'?  I was able to upload a .csv file without issue

**Comment by MingChen0919** [comment]
yes, let me try again.

**Comment by MingChen0919** [comment]
strange, it's working now. 
<img width="251" alt="screen shot 2018-05-30 at 2 07 08 pm" src="https://user-images.githubusercontent.com/1262709/40738922-d1624e90-6412-11e8-9226-a425f9f5e35a.png">

**Comment by heh30** [comment]
For all 4 uploads?  I saw 4 galaxy related jobs on our cluster...

**Comment by MingChen0919** [comment]
yes, i re-uploaded 4 files. it all worked. i am closing this issue.
